### PR TITLE
Update the Norwegian translation

### PR DESCRIPTION
**Pull request by zenseii**

### CODE REVIEW

File: files/lang/nb.po [hunk 1]
```diff
@@ -550,109 +553,106 @@ msgid "Shows the results of individual monster's actions."
...
-msgstr "%{name} bruker %{spell}."
+msgstr "%{name} brukte %{spell}."
 
 msgid "The %{spell} does %{damage} damage to one undead creature."
-msgstr "%{spell} gjør %{value} skadepoeng på én vandød skapning."
+msgstr "%{spell} gjorde %{value} skadepoeng på én vandød skapning."
 
 msgid "The %{spell} does %{damage} damage to all undead creatures."
-msgstr "%{spell} gjør %{damage} skadepoeng på alle vandøde."
+msgstr "%{spell} gjode %{damage} skadepoeng på alle vandøde."
```
Comment (zenseii) [inline files/lang/nb.po:596]:
```suggestion
msgstr "%{spell} gjorde %{damage} skadepoeng på alle vandøde."
```

File: files/lang/nb.po [hunk 2]
```diff
@@ -550,109 +553,106 @@ msgid "Shows the results of individual monster's actions."
...
-msgstr "Troppen med %{attacker} får uflaks."
+msgstr "Troppen med %{attacker} fikk uflaks."
 
 msgid "Good luck shines on the %{attacker}."
-msgstr "Troppen med %{attacker} får flaks."
+msgstr "Troppen med %{attacker} fikk flaks."
 
 msgid "High morale enables the %{monster} to attack again."
-msgstr "God kampånd gjør at troppen med %{monster} får en ny tur."
+msgstr "God kampånd gjode at troppen med %{monster} fikk en ny tur."
```
Comment (zenseii) [inline files/lang/nb.po:639]:
```suggestion
msgstr "God kampånd gjorde at troppen med %{monster} fikk en ny tur."
```

File: files/lang/nb.po [hunk 3]
```diff
@@ -3425,33 +3383,39 @@ msgstr "Umulig"
...
 msgstr ""
+"Velg denne vanskelighetsgraden dersom du foretrekker historie framfor "
+"utfordring. Datamaskinspilleren vil være svakere sammenlignet med middels "
+"vanskelighetsgrad."
 
 msgid ""
 "Choose this difficulty to enjoy the campaign as per the original design."
 msgstr ""
+"Velg denne vanskelighetsgraden for å nyte felttoget slik som det opprinnelig "
+"ble uttenkt."
```
Comment (zenseii) [inline files/lang/nb.po:3401]:
```suggestion
"Velg denne vanskelighetsgraden for å nyte felttoget slik som det opprinnelig "
"var tiltenkt."
```

File: files/lang/nb.po [hunk 4]
```diff
@@ -3464,6 +3428,7 @@ msgstr "Vis filmen for det nåværende punktet av felttoget."
 
 msgid "Select campaign difficulty. It cannot be changed after."
 msgstr ""
+"Velg vanskelighets grad for felttoget. Dette vil ikke kunne forandres senere."
```
Comment (zenseii) [inline files/lang/nb.po:3431]:
```suggestion
"Velg vanskelighetsgrad for felttoget. Dette vil ikke kunne forandres senere."
```

File: files/lang/nb.po [hunk 5]
```diff
@@ -3584,7 +3549,7 @@ msgstr ""
 "Det opprinnelige Heroes of Might and Magic II: The Price of Loyalty-teamet"
 
 msgid "Cyberlore Studios"
-msgstr ""
+msgstr "Cyberlore Studios"
```
Comment (zenseii) [inline files/lang/nb.po:3552]:
```suggestion
msgstr ""
```

File: files/lang/nb.po [hunk 6]
```diff
@@ -3602,7 +3567,7 @@ msgid "Playtesters"
 msgstr "Spilltestere"
 
 msgid "New World Computing"
-msgstr ""
+msgstr "New World Computing"
```
Comment (zenseii) [inline files/lang/nb.po:3570]:
```suggestion
msgstr ""
```

File: files/lang/nb.po [hunk 7]
```diff
@@ -3653,14 +3618,14 @@ msgid "Standard"
 msgstr "Standard"
 
 msgid "View High Scores for Standard Maps."
-msgstr "Vis rekordtavle for standard kart."
+msgstr "Vis rekordtavle for vanlige brett."
 
 # Kampanje?
```
Comment (zenseii) [inline files/lang/nb.po:3623]:
```suggestion
```

File: files/lang/nb.po [hunk 8]
```diff
@@ -4064,11 +4031,12 @@ msgid ""
 "The difficulty rating reflects a combination of various settings for your "
 "game. This number will be applied to your final score."
 msgstr ""
-"Utbyttegraderingen gjenspeiler en sammensetting av de forskjellige valgene "
-"for partiet ditt. Dette tallet vil påvirke den endelige poengsummen din."
+"Utbytteprosenten regnes ut av summen av de forskjellige innstilligene du har "
```
Comment (zenseii) [inline files/lang/nb.po:4034]:
```suggestion
"Utbytteprosenten regnes ut som summen av de forskjellige innstilligene du har "
```

File: files/lang/nb.po [hunk 9]
```diff
@@ -4064,11 +4031,12 @@ msgid ""
...
+"valgt partiet ditt. Denne prosenten påvirker den endelige poengsummen din "
+"dersom du vinner partiet."
```
Comment (zenseii) [inline files/lang/nb.po:4036]:
```suggestion
"valgt for partiet ditt. Denne prosenten påvirker den endelige poengsummen "
"din dersom du vinner partiet."
```

File: files/lang/nb.po [hunk 10]
```diff
@@ -4965,16 +4948,17 @@ msgid ""
 "You've come upon a mercenary camp practicing their tactics. \"You're too "
 "advanced for us,\" the mercenary captain says. \"We can teach nothing more.\""
 msgstr ""
-"Du kommer over en leiesoldatleir hvor det drives kamptrening. \"Du har et "
-"mye høyere nivå enn oss,\" sier leiesoldatenes kaptein. \"Det er ingenting "
-"vi kan lære deg.\""
+"Du kommer over en leiren til noen leiesoldater som driver med kamptrening. "
```
Comment (zenseii) [inline files/lang/nb.po:4951]:
```suggestion
"Du kommer over leiren til noen leiesoldater som driver med kamptrening. "
```

File: files/lang/nb.po [hunk 11]
```diff
@@ -4965,16 +4948,17 @@ msgid ""
...
+"\"Du har et mye høyere nivå enn oss,\" sier leiesoldatenes kaptein. \"Det er "
+"ingenting vi kan lære deg.\""
 
 msgid ""
 "You've come upon a mercenary camp practicing their tactics. The mercenaries "
 "welcome you and your troops and invite you to train with them."
 msgstr ""
-"Du kommer over en leiesoldatleir hvor det drives kamptrening. Leiesoldatene "
-"ønsker deg og styrkene dine velkomne og inviterer dere til å trene med dem."
+"Du kommer over en leiren til noen leiesoldater som driver med kamptrening. "
```
Comment (zenseii) [inline files/lang/nb.po:4959]:
```suggestion
"Du kommer over leiren til noen leiesoldater som driver med kamptrening. "
```

File: files/lang/nb.po [hunk 12]
```diff
@@ -5605,9 +5592,9 @@ msgid ""
 "famous nor diplomatic enough to be admitted to see my master,\" he sniffs. "
 "\"Come back when you think yourself worthy.\""
 msgstr ""
-"Hovmesteren åpner døra og gransker deg opp og ned. \"Du er verken berømt "
-"eller veltalende nok til å få tillatelse til å møte herren min,\" fnyser "
-"han. \"Kom tilbake når du kan anse deg som verdig.\""
+"Hovmesteren åpner døra og skuer deg opp og ned. \"Du er verken berømt eller "
+"veltalende nok til å få tillatelse til å møte herren min,\" fnyser han. "
+"\"Kom tilbake når er verdig.\""
```
Comment (zenseii) [inline files/lang/nb.po:5597]:
```suggestion
"\"Kom tilbake når du er verdig.\""
```

File: files/lang/nb.po [hunk 13]
```diff
@@ -5685,19 +5674,21 @@ msgid ""
...
+"\"Det er godt å se deg, min elev. Jeg håper lærdommen min har hjulpet deg.\""
 
 msgid ""
 "Upon your approach, the tree opens its eyes in delight. \"Ahh, an "
 "adventurer! Allow me to teach you a little of what I have learned over the "
 "ages.\""
 msgstr ""
-"Ved din ankomst åpner treet øynene sine i glede. \"Ahh, en eventyrer! Tillat "
-"meg å lære deg litt av hva jeg har lært gjennom tidene.\""
+"Du ankommer treet og det åpner øynene sine i glede i det du kommer nærmere. "
```
Comment (zenseii) [inline files/lang/nb.po:5685]:
```suggestion
"Du ankommer treet og det åpner øynene sine i glede idet du kommer nærmere. "
```

File: files/lang/nb.po [hunk 14]
```diff
@@ -5685,19 +5674,21 @@ msgid ""
...
+"\"Ahh, en eventyrer! Tillat meg å lære deg litt av hva jeg har lært gjennom "
+"tidene.\""
 
 msgid "Upon your approach, the tree opens its eyes in delight."
-msgstr "Idet du ankommer, åpner treet øynene sine i glede."
+msgstr ""
+"Du ankommer treet og det åpner øynene sine i glede i det du kommer nærmere."
```
Comment (zenseii) [inline files/lang/nb.po:5691]:
```suggestion
"Du ankommer treet og det åpner øynene sine i glede idet du kommer nærmere."
```

File: files/lang/nb.po [hunk 15]
```diff
@@ -5911,8 +5902,9 @@ msgid ""
 "You are now wiser for the visit, and gain %{exp} experience."
 msgstr ""
 "En ulende og nifs sang kommer fra sirenene på svaberget. Flere av mannskapet "
-"ditt blir offer for forhekselsen og stuper ut i sjøen hvor de drukner. "
-"Hendelsen ga deg lærdom, og du får %{exp} erfaringspoeng."
+"ditt blir offer for forhekselsen deres og stuper ut i sjøen hvor de drukner. "
+"Du kommer helberget ut av hendelsen med ny lærdom og får %{exp} "
```
Comment (zenseii) [inline files/lang/nb.po:5906]:
```suggestion
"Du kommer helberget ut av hendelsen med ny lærdom og %{exp} "
```

File: files/lang/nb.po [hunk 16]
```diff
@@ -5976,9 +5967,9 @@ msgid ""
 "you with a paw, knocking you to the ground. Another blow makes the world go "
 "black, and you know no more."
 msgstr ""
-"\"Du gjettet feil\", sier Sfinksen med et lurt smil. Sfinksen denger til deg "
-"med en pote slik at du går rett i bakken. Nok et slag treffer deg, og det "
-"svartner for deg. Mer får du ikke med deg."
+"\"Du gjettet feil\", sier Sfinksen med et lurt smil før den denger til deg "
```
Comment (zenseii) [inline files/lang/nb.po:5970]:
```suggestion
"\"Du gjettet feil\", sier sfinksen med et lurt smil før den denger til deg "
```

File: files/lang/nb.po [hunk 17]
```diff
@@ -6462,8 +6455,8 @@ msgid ""
 "%{skill} gives your hero's catapult shots a greater chance to hit and do "
 "damage to castle walls."
 msgstr ""
-"%{skill} gjør at skuddene til helten din sin katapult får større sjanse til "
-"å treffe og skade ringmuren til en borg."
+"%{skill} gjør at skuddene til katapulten din har større sjanse til å treffe "
```
Comment (zenseii) [inline files/lang/nb.po:6458]:
```suggestion
"%{skill} gjør at skuddene til katapulten din har større sjanse for å treffe "
```

**Comment by zenseii** [comment]
Hi, @ihhub.
This PR is ready for you to have a look at when you should have the time.

**Comment by ihhub** [comment]
@zenseii , thank you so much for the update of this translation!

**Pull request merged**